### DISCUSSION TITLE
Regular docker image updates ❤️

### DISCUSSION THREAD

**Discussion by ThraaxSession** [hoppscotch/hoppscotch#3149] [Kind words]
Hi together,

First, I think it's a great app. But I have issues to have an updated version with docker. Could you creat a process to keep it up to date?

**Comment by jobartim44** [comment]
Do you imagine having regular hoppscotch on docker hub at some point ?

**Reply by ThraaxSession** [reply]
Hi, yes exactly. After each release mapped to latest tag. The docker image is always pretty old.

**Comment by jobartim44** [comment]
I think we should have another way to add our environment variables into the docker image

**Reply by ThraaxSession** [reply]
Any solution is fine, if the docker images are up to date:)

**Comment by ThraaxSession** [comment]
Any updates on docker images? I assume the latest release isn't also available as image?

**Reply by jamesgeorge007** [reply]
Hi, we've been following a monthly release cycle where the images are published to [Docker Hub](https://hub.docker.com/u/hoppscotch).
